oak_hello_world_linux_init/README.md

```diff
@@ -49,14 +49,22 @@ cp --archive target/x86_64-unknown-linux-musl/release/oak_hello_world_linux_init
     > bin/initramfs
 ```
 
+Build the Stage 0 Firmware image:
+
+```bash
+( cd stage0; cargo build --release; )
+objcopy --output-format binary stage0/target/x86_64-unknown-none/release/oak_stage0 \
```

Comment:
I added `cargo-objcopy` to the Docker image, so this can be done in a single step:
`cargo objcopy --release -- -O binary stage0/foo/bar/baz`

(Doing the objcopy separately is not wrong, of course, so feel free to leave it as-is.)

Reply:
I somewhat prefer making the two steps clear as part of this readme, so will leave as is for now.